ide/db/src/org/netbeans/modules/db/explorer/node/ProcedureNode.java

```diff
@@ -657,29 +672,26 @@ public boolean isEditSourceSupported() {
         @Override
         public String getDDL() {
             StringBuilder expression = new StringBuilder();
-            // set delimiter
             expression.append("DELIMITER ").append(DELIMITER).append(NEW_LINE); // NOI18N
-            // DDL
-            expression.append("CREATE OR REPLACE ").append(getSource());
+            expression.append("CREATE OR REPLACE ").append(getSource()); // NOI18N
             expression.append(SPACE).append(DELIMITER).append(NEW_LINE); // NOI18N
-            // unset delimiter
             expression.append("DELIMITER ; ").append(NEW_LINE); // NOI18N
             return expression.toString();
         }
 
         private String fqn(String source, String owner) {
             String upperSource = source.toUpperCase();
-            String toFind = getTypeName(getType()) + " "; // NOI18N
+            String toFind = getTypeName(getType()) + " ";
             String res = source;
             int nameIdx = upperSource.indexOf(toFind);
             if (nameIdx != -1) {
                 // don't duplicate owner
-                if (upperSource.substring(nameIdx + toFind.length()).trim().startsWith(owner.toUpperCase() + '.')) { // NOI18N
+                if (upperSource.substring(nameIdx + toFind.length()).trim().startsWith(owner.toUpperCase() + '.')) {
                     return source;
                 }
                 res = source.substring(0, nameIdx + toFind.length()) +
                         owner +
-                        '.' + // NOI18N
+                        '.' +
```

Comment:
//NOI18N was dropped

Reply:
Perhaps, I suppose we need not add it for char, but basically, you should leave "existing" `// NOI18N`s as it is because it is not related to the issue.